tests/test_customization.py

```
@@ -11,54 +11,49 @@
-    with patch("unraid_mcp.tools.customization.make_graphql_request", new_callable=AsyncMock) as m:
+    with patch("unraid_mcp.tools.unraid.make_graphql_request", new_callable=AsyncMock) as m:
         yield m
 
 
-def _make_tool():
-    return make_tool_fn(
-        "unraid_mcp.tools.customization",
-        "register_customization_tool",
-        "unraid_customization",
-    )
+def _make_tool() -> Any:
```

Ruff (0.15.5)

[error] Undefined name `Any`

(F821)